lib/utilities/validateRange.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Validates numbers ranges to not exceed 100 entities
+ *
+ * @param {*} value  The value to validate
```

Comment:
Given that you already expect a string (based on the DTO function), and you expect it to be in a certain shape with certain separators, perhaps you can add the type and details in docs